src/Generator/Generator.csproj

```diff
@@ -43,7 +43,8 @@
 
   <ItemGroup>
     <Using Include="Azure.DigitalTwins.Core" />
-    <Using Include="Microsoft.Azure.DigitalTwins.Parser" />
+    <Using Include="DTDLParser.Models" />
```

Comment:
The new parser does not use MS.Azure namepsace but is using DTDL namespace? Or this is completely a different one.

Reply:
The existing package Microsoft.Azure.DigitalTwins.Parser has been deprecated and they have recommended to use the DTDLParser.Models package for V2 & V3 Models.